src/ceph-disk/ceph_disk/main.py

```diff
@@ -1,12 +1,10 @@
 #!/usr/bin/env python
 #
-# Copyright (C) 2015, 2016 Red Hat <contact@redhat.com>
+# Copyright (C) 2015-2016 Red Hat <contact@redhat.com> Authors
 # Copyright (C) 2014 Inktank <info@inktank.com>
 # Copyright (C) 2014 Cloudwatt <libre.licensing@cloudwatt.com>
 # Copyright (C) 2014 Catalyst.net Ltd
 #
-# Author: Loic Dachary <loic@dachary.org>
-#
```

Comment:
Please keep this line (the rationale is explained at http://tracker.ceph.com/issues/16125#note-10)